src/Microsoft.DotNet.ProjectModel/WorkspaceContext.cs

```diff
@@ -280,6 +351,20 @@ private FileModelEntry<LockFile> AddLockFileEntry(string projectDirectory, FileM
             return currentEntry;
         }
 
+        private ProjectContextBuilder InitializeProjectContextBuilder(Project project)
+        {
+            var builder = new ProjectContextBuilder()
+                .WithProjectResolver(path => GetProjectCore(path)?.Model)
```

Comment:
this doesn't feel right. when path can't be normalized that is when the project returned is null. then the builder will call a project resolver which always return null. doesn't it fail anything? we have test case cover it? 

i think the flow should end here, the `GetProjectCore` shouldn't return `null` at all.